### PR TITLE
Add black and updated docs

### PR DESCRIPTION
**Pull request by felipem775**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -4,3 +4,5 @@ coverage==5.3
 pylint==2.6.0
 pytest-cov==2.10.1
 pylint==2.6.0
+pre-commit==2.7.1
+black==20.8b1
```
Comment (avara1986) [inline requirements-dev.txt:8]: Add this packages in Pipfile too

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[tool.black]
```
Comment (avara1986) [inline pyproject.toml:1]: It's ok the `pyproject.toml` but, in the **microservices-scaffold**, this configuration could be placed in `setup.cfg` and this file contains at this moment the configuration of the linters and other libaries, could be better put all the configuration in a single file :+1:

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+repos:
+        - repo: https://github.com/ambv/black
```
Comment (avara1986) [inline .pre-commit-config.yaml:2]: replace these spaces for [2 paces](https://docs.saltstack.com/en/master/topics/troubleshooting/yaml_idiosyncrasies.html#:~:text=The%20suggested%20syntax%20for%20YAML,uniform%20and%20not%20deeply%20nested.):

*The suggested syntax for YAML files is to use 2 spaces for indentation, but YAML will follow whatever indentation system that the individual file uses. Indentation of two spaces works very well for SLS files given the fact that the data is uniform and not deeply nested.*

**Review by avara1986 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (avara1986) [reply pyproject.toml:1]: Sorry, my fault, Black not support setup.cfg:

- https://github.com/psf/black/issues/683
- https://github.com/flying-sheep/awesome-python-packaging

**Review by avara1986 — APPROVED** [review]

**Comment by alexppg** [comment]
What do you thing about adding `pytest-black` to the CI to make sure everybody uses it? https://pypi.org/project/pytest-black/ @avara1986 @felipem775

**Comment by avara1986** [comment]
Good idea @alexppg! but we could add this package in these steps:
- Add black with pre-commit
- Update all repository with black's changes
- Add pytest-black to prevent new "errors" of format

What do you say? :) do we create a issue to add this validation? :muscle:

**Comment by alexppg** [comment]
Nice, done then. https://github.com/python-microservices/microservices-scaffold/issues/218 and https://github.com/python-microservices/microservices-scaffold/issues/219

**Review by alexppg — APPROVED** [review]

**Pull request merged**